### PR TITLE
Fix small issues around the animated ItemsRepeater sample

### LINKED ISSUE #579: Narrator does not announce the colors getting selected in ItemsRepeater.
**Describe the bug**

**Steps to reproduce the bug**

1. Turn on Narrator
2. Launch XAML Controls Gallery.
3. Activate ItemsRepeater from home page, navigate to header "Animated Scrolling and Content Display" and select the colors using "caps+arrow" keys and observe.

**Expected behavior**

Narrator should announce the information whether the item got selected or not along with describing other attributes.

**Actual behavior**

Narrator only announces the button name when activating it but tells nothing whether it is selected or not.

**Screenshots**

image

**Version Info**

Gallery version (found on Settings page): 1.2.13.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/25440841)

### PR DESCRIPTION
**Pull request by chingucoding**
## Description

Fix issue where the selected element was not indicated to users using assistive technology.

Also switched to ViewChanged event as the selected color was faulty with the ViewChanging event.

## Motivation and Context

Closes #579 
## How Has This Been Tested?

Test manually
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 1]
```diff
@@ -291,6 +299,17 @@ private void Animated_ScrollViewer_ViewChanging(object sender, ScrollViewerViewC
...
         }
 
+        private void SetUIANamesForSelectedEntry(Button selectedItem)
+        {
+            if (LastSelectedColorButton != null && LastSelectedColorButton.Content is string content)
+            {
+                AutomationProperties.SetName(LastSelectedColorButton, content);
+            }
+
+            AutomationProperties.SetName(selectedItem, (string)selectedItem.Content + " , selected");
```
Comment (anawishnoff) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs:309]: Do we need a null check for selected item?

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 2]
```diff
@@ -279,9 +284,12 @@ private void GetButtonSize(object sender, RoutedEventArgs e)
             AnimatedBtnMargin = AnimatedBtn.Margin;
         }
 
-        private void Animated_ScrollViewer_ViewChanging(object sender, ScrollViewerViewChangingEventArgs e)
+        private void Animated_ScrollViewer_ViewChanged(object sender, ScrollViewerViewChangedEventArgs e)
```
Comment (anawishnoff) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs:287]: Why the change (not against it, just curious)?

Comment (chingucoding) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs:287]: ViewChanging does not fire once scrolling is done but rather in between. So if you scroll fast, the item in center and the displayed color are actually different (sometimes it isn't even on the screen anymore):

Selected color not matching the visually selected color

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs (hunk 1 above)
Comment (chingucoding) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs:309]: You could but the code would fail on different areas if the passed parameter would be null so there is no benefit in adding that I think.

**Review by anawishnoff — APPROVED** [review]
Same conflict issue as the other PR - otherwise approved!
